plotly/tests/test_optional/test_offline/test_offline.py

```diff
@@ -0,0 +1,31 @@
+"""
+test__offline
+
+"""
+from __future__ import absolute_import
+from nose.tools import raises
+import os
+
+
+from plotly.exceptions import PlotlyError
+import plotly
+
+
+@raises(PlotlyError)
```

Comment:
ditto in here, re: https://github.com/plotly/python-api/pull/236/files#r33808672